src/containers/AyatulKursi/index.js

```diff
@@ -50,9 +50,9 @@ const TopOptions = Loadable({
 });
 
 const title = 'Ayatul Kursi';
-
-const description =
-  'Ayatul Kursi is verse 255 of the second chapter, surah Baqarah of the Holy Quran, Surah al-Baqarah (The Chapter of the Cow).  It is also known as the Throne Verse.';
+const description = 'Ayatul Kursi is verse 255 of the second chapter in the ' +
+                    'Holy Quran, Surah al-Baqarah (The Chapter of the Cow). ' +
```

Comment:
`second chapter, Surah al_Baqarah( The Chapter of Cow) of Holy Quran` feel more readable :)